index.html

```diff
@@ -2287,6 +2412,16 @@ <h2 class="code-line" data-line-start=9 data-line-end=10><a id="Companies_that_r
 					<td>Referral <a href="https://t.me/nukich74">@nukich74</a></td>
 					<td>07.04.22</td>
 				</tr>
+				<tr>
+					<td style="text-align:center"><a href="https://www.pipedrive.com/jobs/open-positions?location=estonia">Pipedrive</a></td>
+					<td style="text-align:center"><i class="em em-flag-ee" aria-role="presentation" aria-label="Estonia Flag"></i>
+					</td>
+					<td style="text-align:center">Tallinn, Tartu</td>
+					<td style="text-align:center">Relocates</td>
+					<td style="text-align:center">Still relocates</td>
```

Comment:
As far as I know, Estonia has frozen new work visas unfortunately

Reply:
Please confirm the status is correct one or provide the correct wording for this case :)

Reply:
You got status correctly, closing this